mkosi/backend.py

```diff
@@ -168,6 +168,19 @@ def __new__(cls, number: int, package_type: PackageType) -> Distribution:
     def __str__(self) -> str:
         return self.name
 
+def is_rpm_distribution(d: Distribution) -> bool:
+    return d in [
+        Distribution.fedora,
+        Distribution.mageia,
+        Distribution.centos,
+        Distribution.centos_epel,
+        Distribution.openmandriva,
+        Distribution.rocky,
+        Distribution.rocky_epel,
+        Distribution.alma,
+        Distribution.alma_epel
+    ]
```

Comment:
```suggestion
    return d in (
        Distribution.fedora,
        Distribution.mageia,
        Distribution.centos,
        Distribution.centos_epel,
        Distribution.openmandriva,
        Distribution.rocky,
        Distribution.rocky_epel,
        Distribution.alma,
        Distribution.alma_epel
    )
```
not that it really matters, but tuples are faster.